share/spice/github/triggers.txt

```diff
@@ -0,0 +1,315 @@
+actionscript
+c#
+c++
+clojure
+coffeescript
+common lisp
+css
+c
+diff
+emacs lisp
+erlang
+haskell
+html
+javascript
+java
+lua
+objective-c
+perl
+php
+python
+ruby
+scala
+scheme
+sql
+abap
+ada
+agda
+ags script
+alloy
+ant build system
+antlr
+apacheconf
+apex
+apl
+applescript
+arc
+arduino
+asciidoc
+aspectj
+asp
+assembly
+ats
+augeas
+autohotkey
+autoit
+awk
+batchfile
+befunge
+bison
+blitzbasic
+blitzmax
+bluespec
+boo
+brainfuck
+brightscript
+bro
+c-objdump
+c2hs haskell
+cap'n proto
+ceylon
+chapel
+chuck
+cirru
+clean
+clips
+cmake
+cobol
+coldfusion
+coldfusion cfc
+component pascal
+coq
+cpp-objdump
+creole
+crystal
+cucumber
+cuda
+cycript
+cython
+d-objdump
+darcs patch
+dart
+dm
+dockerfile
+dogescript
+dylan
+d
+e
+eagle
+ecere projects
+ecl
+ec
+edn
+eiffel
+elixir
+elm
+emberscript
+f#
+factor
+fancy
+fantom
+fish
+flux
+forth
+fortran
+frege
+g-code
+game maker language
+gams
+gap
+gas
+gdscript
+genshi
+gentoo ebuild
+gentoo eclass
+gettext catalog
+glsl
+glyph
+gnuplot
+golo
+gosu
+go
+grace
+grammatical framework
+graph modeling language
+graphviz (dot)
+groff
+groovy server pages
+groovy
+hack
+haml
+handlebars
+harbour
+haxe
+html+django
+http
+hy
+idl
+idris
+igor pro
+inform 7
+ini
+inno setup
+ioke
+io
+irc log
+isabelle
+jade
+java server pages
+json5
+jsoniq
+jsonld
+json
```

Comment:
I feel like JSON, HTTP and similar should be removed from this list. It's more likely that someone is looking for a JSON parser in another language/platform then code written in JSON.